lib/backpex/fields/textarea.ex

```diff
@@ -34,6 +34,7 @@ defmodule Backpex.Fields.Textarea do
         <BackpexForm.input
           type="textarea"
           field={@form[@name]}
+          rows={@field_options[:rows] || 3}
```

Comment:
```suggestion
          rows={@field_options[:rows] || 2}
```

According to the mdn web docs, the row attribute defaults to 2. so let's use 2 for the default, too.

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#rows

Reply:
Perfect 🤣